setup.cfg

```diff
@@ -49,6 +49,12 @@ install_requires =
     wikipedia2vec
     sentence-transformers
     nltk
+    transformers
+    sklearn
+    json
+    tensorflow_hub
+    numpy
```

Comment:
All of these package dependencies should have version pins (if not exact, then using the pessimistic `~=` operator) in order to avoid the [dependency confusion security vulnerability](https://medium.com/@alex.birsan/dependency-confusion-4a5d60fec610).